venter/migrations/0004_auto_20180923_1236.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.0.7 on 2018-09-23 07:06
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('venter', '0003_auto_20180920_1625'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='comment',
+            options={'verbose_name': 'Comment', 'verbose_name_plural': 'Comments'},
```

Comment:
This migration doesn't do anything

Reply:
In the initial migration, there it is mentioned in the options of the comment model that it is being ordered in '-time', I have just removed it